components/admin_console/workspace-optimization/cta_buttons.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {useHistory} from 'react-router-dom';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import {TELEMETRY_CATEGORIES} from 'utils/constants';
+
+import './dashboard.scss';
+
+type CtaButtonsProps = {
+    learnMoreLink: string;
+    learnMoreText: string;
+    actionLink: string;
+    actionText: React.ReactNode;
+    telemetryAction?: string;
+    actionButtonCallback?: () => void;
+};
```

Comment:
I am ok with how it is built ... just one suggestion: We might not always have both buttons, so we probably should make the text and link properties optional and just return `null` in a guard for the buttons itself. WDYT?

Reply:
ok, if there is that scenario where we don't have both buttons, then I guess we should make all the params that create buttons optionals. I went with this initial approach because I didn't see any example/scenario where we didn't use both buttons.